include/stout/flags/flags.cc

```diff
@@ -633,6 +624,22 @@ void Parser::Parse(const std::vector<ArgumentInfo>& values) {
     if (parsed_.count(pos_arg.field) > 0)
       continue;
 
+    const std ::string normalized_value =
+        (pos_arg.field->type()
+         != google::protobuf::FieldDescriptor::TYPE_STRING)
+        ? argument.default_()
```

Comment:
You have similar logic to normalize a string value on line 573 above.

Consider extracting a helper that's used in both places.